go/test/endtoend/cluster/cluster_process.go

```diff
@@ -400,6 +420,7 @@ func (cluster *LocalProcessCluster) GetVtgateInstance() *VtgateProcess {
 // NewCluster instantiates a new cluster
 func NewCluster(cell string, hostname string) *LocalProcessCluster {
 	cluster := &LocalProcessCluster{Cell: cell, Hostname: hostname}
+	go cluster.CtrlCHandler()
```

Comment:
CtrlCHandler will only work for the testcase which uses the NewCluster function to create new cluster.
it wont work where cluster is created directly like
	`clusterInstance = &cluster.LocalProcessCluster{Cell: cell, Hostname: hostname}`